server/plugins/encryption/encryption.go

```diff
@@ -62,56 +65,17 @@ func (b builder) Build() {
 	keyType := b.detectKeyType()
 
 	if !enabled && (disableFlag || keyType == keyTypeNone) {
-		noEncryptionBuilder{}.WithClients(b.clients).Build()
+		_, err := noEncryptionBuilder{}.WithClients(b.clients).Build()
+		if err != nil {
+			log.Fatal().Err(err).Msg("failed initializing server in unencrypted mode")
```

Comment:
this should also return error and log error in: https://github.com/inDriver/woodpecker-encrypted/blob/39287b9932837a0be0c424005430f3f8e95034d6/cmd/server/server.go#L267

Reply:
will fix now. and found another 2 fatals in wrapper